src/lib/Transpose.hs

```diff
@@ -256,6 +256,8 @@ transposeMiscOp op _ = case op of
   BitcastOp    _ _      -> notImplemented
   UnsafeCoerce _ _      -> notImplemented
   GarbageVal _          -> notImplemented
+  ShowAny _ -> error "Shouldn't have ShowAny in simplified IR"
+  ShowScalar _ -> error "Shouldn't have ShowScalar in simplified IR"
```

Comment:
We can't use IR tags to prove this doesn't happen?  Seems like an obvious use case for them...

Reply:
Yes, absolutely. That will happen as part of #1188 .